### PR TITLE
Remove the ability to create new explainer

### PR DESCRIPTION
**Pull request by mchv**
Remove the ability in the UI to create new explainer.
This the first step of the deprecation of the tool

### CODE REVIEW

File: explainer-server/app/views/explainList.scala.html [hunk 1]
```diff
@@ -46,12 +46,6 @@
                             </select>
                         </label>
                     </li>
-                    <li class="nav__item top-toolbar__item top-toolbar__item--no-spacing">
-                        <button class="top-toolbar__button"
-                           type="button" onclick="views.ExplainList().createNewExplainer()">
```
Comment (regiskuckaertz) [inline explainer-server/app/views/explainList.scala.html:51]: Shouldn't `ExplainList` and `createNewExplainer` be removed too, then?

Comment (mchv) [reply explainer-server/app/views/explainList.scala.html:51]: At the end we will shutdown the entire project, so only preventing users to create new ones from the time being.

Comment (regiskuckaertz) [reply explainer-server/app/views/explainList.scala.html:51]: oh yeah, that makes total sense 🚀

**Comment by tomrf1** [comment]
👍

**Pull request merged**